### PR TITLE
add docs for distributed tracing

### PR DESCRIPTION
**Pull request by gernest**
Part of #854

### CODE REVIEW

**Comment by buger** [comment]
@gernest can you add a bit more "educational" part here? What is distributed tracing, why it is needed, and how our users can leverage it.

**Review by marksou — APPROVED** [review]
Made a few minor amendments.

File: tyk-docs/content/distributed-tracing/distributed-tracing.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Supported tracers
+- [Jaeger](https://www.jaegertracing.io/)
+- [Zipkin](https://zipkin.io/)
+
+## Enabling distributed tracing
+To enable distributed tracing, add the following tracing configuration on your `tyk.conf`
+
+```.json
+    "tracing": {
+        "name": "${tracer_name}",
```
Comment (asoorm) [inline tyk-docs/content/distributed-tracing/distributed-tracing.md]: I would reduce indentation. + for json, 2-space tabs.

File: tyk-docs/content/distributed-tracing/jaeger.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Jaeger"
+date: 2019-07-29T10:28:52+03:00
+weight: 1
+menu: 
+  main:
+    parent:  "Distributed Tracing"
+---
+
+Jaeger is a distributed tracing system.It is used for monitoring and troubleshooting microservices-based distributed systems. To lean more about jaeger [visit their website](https://www.jaegertracing.io/)
```
Comment (asoorm) [inline tyk-docs/content/distributed-tracing/jaeger.md]: `stem.It` Space required after full-stop.

File: tyk-docs/content/distributed-tracing/jaeger.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+To enable this tracer, you need to have a working jaeger server.
+
+## Configuring
+
+In `tyk.conf` on `tracing` setting
+
+```
+    "tracing": {
```
Comment (asoorm) [inline tyk-docs/content/distributed-tracing/jaeger.md]: Formatting of json. as per comment above.

File: tyk-docs/content/distributed-tracing/jaeger.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        "options": {}
+    }
+```
+
+`options` are settings that are used to initialize the jaeger client. For more details about the options [see client libraries](https://www.jaegertracing.io/docs/1.11/client-libraries/)
+
+# Sample configuration
+
+```json
+    "tracing": {
```
Comment (asoorm) [inline tyk-docs/content/distributed-tracing/jaeger.md]: formatting

File: tyk-docs/content/distributed-tracing/zipkin.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+Zipkin is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in service architectures.
+
+To enable this tracer, you need to have a working zipkin server.
+
+## Configuring
+
+In `tyk.conf` on `tracing` setting
+
+```
+    "tracing": {
```
Comment (asoorm) [inline tyk-docs/content/distributed-tracing/zipkin.md]: formatting + syntax highlight annotation missing

File: tyk-docs/content/distributed-tracing/zipkin.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        "options": {}
+    }
+```
+
+`options` are settings that are used to initialize the zipkin client.
+
+# Sample configuration
+
+```json
+    "tracing": {
```
Comment (asoorm) [inline tyk-docs/content/distributed-tracing/zipkin.md]: formatting

**Review by asoorm — CHANGES_REQUESTED** [review]
minor nits

**Review by asoorm — APPROVED** [review]
LGTM - Agree with @buger about educational piece

**Pull request merged**